modules/tfm/tfm/boards/CMakeLists.txt

```diff
@@ -25,8 +25,21 @@ target_include_directories(platform_region_defs INTERFACE ${partition_includes})
 
 target_include_directories(platform_s
   PUBLIC ${partition_includes} ${board_includes}
+  ${NRF_DIR}/include
   )
 
+if (${TFM_PARTITION_CRYPTO} AND (NOT ${PLATFORM_DUMMY_CRYPTO_KEYS}))
```

Comment:
You can now check the CRYPTO_MODULES, I guess some of them must be enabled for these features to be required?

https://github.com/nrfconnect/sdk-zephyr/blob/master/modules/trusted-firmware-m/Kconfig#L204

Reply:
Added check on CRYPTO_KEY_DERIVATION_MODULE_DISABLED